### ISSUE TITLE
PHPUnit 9 compatibility

### ISSUE THREAD

**Issue by galbus** [wp-graphql/wp-graphql-testcase#19]
When I attempt to `composer require` this package in my PHPUnit 9 project it fails with the following dependency/version error:

```
    - Root composer.json requires wp-graphql/wp-graphql-testcase ^2.1.3 -> satisfiable by wp-graphql/wp-graphql-testcase[v2.1.3].
    - wp-graphql/wp-graphql-testcase v2.1.3 requires phpunit/phpunit ^7.5 || ^8.5.0 -> found phpunit/phpunit[7.5.0, ..., 7.5.20, 8.5.0, ..., 8.5.26] but it conflicts with your root composer.json require (^9.5).
```

See [Supported PHPUnit versions](https://phpunit.de/supported-versions.html) for PHPUnit's release timeline.

**Comment by justlevine** [comment]
Issue is that `TraversableContains` becomes an abstract in 9.x so we'd need a shim for `TraversableContainsIdentical` to maintain compat with 8.5 and below. 
I'd offer, but phpunit is not my wheelhouse.

**Comment by kidunot89** [comment]
@galbus @justlevine This issue should be fixed with the release of `v2.2.0`

**Comment by justlevine** [comment]
@kidunot89 you sure ?

![image](https://user-images.githubusercontent.com/29322304/170805623-53e3ee07-553a-42e5-bc09-43c3b0dd786c.png)
Its also causing WPGraphQL's core GH actions [to fail](https://github.com/wp-graphql/wp-graphql/runs/6633938219?check_suite_focus=true) (since `wp-graphql-testcase` was set to `~2.1`, it gets updated to 2.3 by the [docker entrypoint](https://github.com/wp-graphql/wp-graphql/blob/8fcb6dbb59807c6a917bf2e3217489df43c3c301/docker/testing.entrypoint.sh#L89) )

cc: @jasonbahl
